CDP4-SDK-Samples/SimpleConsoleApp.cs

```diff
@@ -0,0 +1,507 @@
+    using NLog.Targets;
+
+    public class SimpleConsoleApp
+    {
+        private ISession session;
+        private Credentials credentials;
+        private bool isRunning = true;
+        private Uri uri;
+
+        // Commands
+        private const string openConnection = "open";
+        private const string refreshData = "refresh";
+        private const string reloadData = "reload";
+        private const string closeConnection = "close";
+        private const string restoreServer = "restore";
+        private const string openIteration = "get_iteration";
+        private const string postPredifinedPerson = "post_person";
+        private const string postPredifinedParameter = "post_parameter";
+        private const string postPfsl = "post_pfsl";
+        private const string postPfslReorder = "post_pfsl_reorder";
+        private const string removeParameterFromElementDefinition = "remove_parameter";
+
+        public void Run()
+        {
+            // Setup NLog
+            var config = new LoggingConfiguration();
+
+            // Targets where to log to: Console
+            var logConsole = new ConsoleTarget("logConsole");
+
+            // Rules for mapping loggers to targets            
+            config.AddRule(LogLevel.Trace, LogLevel.Fatal, logConsole);
+
+            // Apply config           
+            LogManager.Configuration = config;
+
+            printSeparator();
+            Console.WriteLine("Welcome to CDP4 SDK sample app!");
+            printSeparator();
+
+            Console.WriteLine("Enter your user name (default is admin, just press Enter):");
+            var userName = String.IsNullOrWhiteSpace(Console.ReadLine()) ? "admin" : Console.ReadLine();
+
+            Console.WriteLine("Enter your password (to use default just press Enter):");
+            var pass = String.IsNullOrWhiteSpace(Console.ReadLine()) ? "pass" : Console.ReadLine();
+
+            Console.WriteLine(
+                "Enter a server's URL for future requests (default is https://cdp4services-test.rheagroup.com, just press Enter):");
+            uri = new Uri(String.IsNullOrWhiteSpace(Console.ReadLine())
+                ? "https://cdp4services-test.rheagroup.com"
+                : Console.ReadLine());
+
+            var dal = new CdpServicesDal();
+            credentials = new Credentials(userName, pass, uri, null);
+            session = new Session(dal, credentials);
+
+            printCommands();
+
+            while (isRunning)
+            {
+                try
+                {
+                    executeCommand(Console.ReadLine());
+                }
+                catch (Exception ex)
+                {
+                    printSeparator();
+                    Console.WriteLine("Something went wrong. Sorry about that.");
+                    Console.WriteLine(ex.Message);
+                    printSeparator();
+                }
+            }
+        }
+
+        private void executeCommand(string command)
```

Comment:
Yeah so adding to what @lxatrhea already said on naming conventions. Methods should have PascalCase -> `ExecuteCommand`.